pkg/test/expectations/expectations.go

```diff
@@ -561,10 +561,12 @@ func ExpectMakeNodesAndNodeClaimsInitializedAndStateUpdated(ctx context.Context,
 	ExpectMakeNodeClaimsInitialized(ctx, c, nodeClaims...)
 
 	// Inform cluster state about node and nodeclaim readiness
-	for _, n := range nodes {
+	for i := range nodes {
+		n := nodes[i]
```

Comment:
Not a big deal, mainly just curious why we chose to change this

Reply:
mistaken change for testing, will move it back 👍